README.md

```diff
@@ -1,2 +1,77 @@
-# holdshelf-poller
-Polls Sierra for items placed on a holdshelf. Notifies patrons.
+# Holdshelf Poller
+
+This is an app to poll Sierra for items placed on a holdshelf and notify those patrons.
+
+## Purpose
+
+This app connects to the Sierra database directly to identify items placed on a holdshhelf. The specific holdshelves and item holding locations are configurable. When it encounters an item on a holdshelf, it checks Redis to determine if the hold has already been processed. If it has not been processed, a call is made to the PatronServices Notify endpoint (TK).
```

Comment:
Nope I was just curious -- I remember you telling me these days S3 is preferred for the BIC pollers, so I wanted to know why Redis is better in this case

Reply:
I see. Yeah I think for pollers that run every N minutes and need to remember some token to resume polling, S3 is a fine option because there isn't a need to record much data nor retrieve it very frequently. If this app needs to record a polling timestamp, I'll prob use S3. For storing a bit of info about each record polled - for thousands of records - I think S3 may get messy.